utils/generate_grammar_tables.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python
+"""Generates various info tables from SPIR-V JSON grammar."""
+
+from __future__ import print_function
+
+import functools
+import re
+import simplejson as json
+
+# Prefix for all C variables generated by this script.
+PYGEN_VARIABLE_PREFIX = '_pygen_variable'
+
+CAPABILITY_BIT_MAPPING = {
```

Comment:
Shouldn't this table come from the JSON file?   Otherwise this script is a second source of truth for the capabilities.